mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -360,6 +372,32 @@ def clean(self):
                 )
         return super().clean()
 
+    def _clean_conditional_fields(self):
+        # validate conditional subfields controlled by checkboxes
+        for conditional_checkbox_field, conditional_subfields in self.conditional_fields.items():
+            if not self.cleaned_data.get(conditional_checkbox_field):
```

Comment:
Python question: This is equivalent to the code below right?:

```python
if conditional_checkbox_field not in self.cleaned_data:
```

Reply:
it's not the same. `.get` also looks up the value so this line is more like:
```
if conditional_checkbox_field not in self.cleaned_data or not self.cleaned_data[conditional_checkbox_field]:
```

Reply:
Ah OK, good point, that value may be in the `cleaned_data` dictionary but it may be `False` 👌